main/src/main/scala/sbt/Main.scala

```diff
@@ -119,9 +119,11 @@ private[sbt] object xMain {
   ): (Option[BootServerSocket], Option[Exit]) =
     try (Some(new BootServerSocket(configuration)) -> None)
     catch {
-      case _: ServerAlreadyBootingException
+      case e: ServerAlreadyBootingException
           if System.console != null && !ITerminal.startedByRemoteClient =>
-        println("sbt server is already booting. Create a new server? y/n (default y)")
+        println(
+          s"sbt thinks that server is already booting because of this exception:\n${e.getCause}\nCreate a new server? y/n (default y)"
```

Comment:
Getting the e.getCause.getMessage (assuming neither of getCause and .getMessage return null), might provide a bit more detail to help identify the exact issue (for example, access denied to what?)

Reply:
I think `toString` gives us this info
https://docs.oracle.com/javase/7/docs/api/java/lang/Throwable.html#toString()